coding-the-legislation/10_basic_example.md

```diff
@@ -61,6 +61,54 @@ To make sure that the formula you have just written works the way you expect, yo
 
 You can check the [YAML tests documentation](writing_yaml_tests.md) to learn more about how to write YAML tests, and how to run them.
 
+## Example with Enum variable
+
+Let's say we have a `HOUSING_OCCUPANCY_STATUS` enumeration:
+
+```py
+HOUSING_OCCUPANCY_STATUS = Enum([
+    u'Tenant',
+    u'Owner',
+    u'Free lodger',
+    u'Homeless'])
+```
+
+To declare an enumeration variable `housing_occupancy_status` of `HOUSING_OCCUPANCY_STATUS` in your legislation, the `EnumCol` type should be defined as follow:
+
+```py
+class housing_occupancy_status(Variable):
+    column = EnumCol(
+        enum = HOUSING_OCCUPANCY_STATUS
+        )
+    entity = Household
+    definition_period = MONTH
+    label = u"Legal housing situation of the household concerning their main residence"
+```
+
+A default value could also be added:
+```py
+    column = EnumCol(
+        enum = HOUSING_OCCUPANCY_STATUS,
+        default = 1
+        )
+```
+
+Thus, to get this variable for a given `month` you would call `household('housing_occupancy_status', month)`. Its value is an index of `HOUSING_OCCUPANCY_STATUS` Enum.
+Nevertheless, in a YAML test, favor its string definition as it is more readable than the enumeration index:
```

Comment:
The index is sometimes used in YAML tests (see [zone_apl variable in this yaml test](https://github.com/openfisca/openfisca-france/blob/8598eb580b257c0003df23a22276260bd96831a5/tests/formulas/aides_logement_primo_accedant.yaml)).